pkgs/development/python-modules/ovirt-engine-sdk/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchPypi, pkgs }:
+
+buildPythonPackage rec {
+  pname = "ovirt-engine-sdk-python";
+  version = "4.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hz7rcz7l391n8ixcwvqw3iq5bx26fk9l5c6xxymv036zm3r9pnh";
+  };
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

Reply:
honestly, I just copy-pasted this derivation from some other package. Not sure what the tests do but I can enable them